src/types/tokens.ts

```diff
@@ -0,0 +1,25 @@
+import { 
+    TokenInstanceDetailDto,
+    TokenInstanceDto, 
+    TokenInstanceRequestDto, 
+    UpdateTokenInstanceRequest 
+} from "./openapi";
+import { 
+    AttributeRequestModel, 
+    AttributeResponseModel 
+} from "./attributes";
+
+export type TokenRequestDto = TokenInstanceRequestDto;
+export type TokenRequestModel = Omit<TokenRequestDto, "attributes | customAttributes"> & { attributes: Array<AttributeRequestModel>, customAttributes?: Array<AttributeRequestModel> };
```

Comment:
customAttributes are mandatory in original Dto, please keep it same in model

Reply:
changes updated